cli/generic/forms/fields.py

```diff
@@ -35,6 +37,63 @@ def _deserialize(self, value, attr, obj, **kwargs):
         return address
 
 
+class FamilyNameField(fields.Field):
+    """
+    Implements validation of the family name.
+
+    References:
+        - https://marshmallow.readthedocs.io/en/3.0/custom_fields.html
+    """
+
+    def _deserialize(self, value, attr, obj, **kwargs):
+        """
+        Validate data (family name) that was passed to field.
+        """
+        if value not in FAMILY_NAMES:
+            raise ValidationError(f'The following family name `{value}` is invalid.')
+
+        return value
+
+
+class TransactionIdentifierListField(fields.Field):
+    """
+    Implements validation of the list identifiers.
```

Comment:
`list identifiers` for me means identifier of the list, but not a list of identifiers. What do you think?

Reply:
@dmytrostriletskyi, done.